client/my-sites/invites/invite-accept-logged-out/style.scss

```diff
@@ -48,6 +50,8 @@ body.is-section-accept-invite {
 		padding: 0;
 		margin: 0;
 		max-width: 372px;
+		background: none;
```

Comment:
Flyby fix: there's a barely visible background difference in the logged-out invite form (#fff vs #fdfdfd).

Reply:
Thanks for fixing this.